### PR TITLE
update docs and arg names for ParallelCopyGCSDirectoryIntoHDFSSparkIn…

### PR DESCRIPTION
**Pull request by cwhelan**
…tegrationTest

### CODE REVIEW

**Comment by cwhelan** [comment]
@droazen @lbergelson would one of you want to give this tool doc update a quick review for https://github.com/broadinstitute/gatk/issues/3853?

**Comment by cwhelan** [comment]
Hang on, it looks like this work might have been redundant with changes in https://github.com/broadinstitute/gatk/pull/3917/files.

File: src/main/java/org/broadinstitute/hellbender/tools/spark/ParallelCopyGCSDirectoryIntoHDFSSpark.java [hunk 1]
```diff
@@ -26,17 +26,47 @@
...
+ * <ul>
+ * <li>A path to a file or directory in Google Cloud Storage. If the path indicates a directory it <em>must</em> include a
+ * trailing slash.</li>
+ * <li>A path to a new directory in the HDFS file system to which the data will be copied. This directory will be created by
+ * the tool and populated with the data transferred from GCS.</li>
+ * </ul>
+ *
+ * <h3>Usage example</h3>
+ * <pre>
+ *     ./gatk ParallelCopyGCSDirectoryIntoHDFSSpark \
```
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/spark/ParallelCopyGCSDirectoryIntoHDFSSpark.java]: ./gatk --> gatk

**Comment by sooheelee** [comment]
Hey @cwhelan I made a minor change `./gatk` --> `gatk`.

**Comment by droazen** [comment]
@sooheelee Note that for many users, `./gatk` will work, while just `gatk` will not. It depends on the PATH setup in their shell.

File: scripts/spark_eval/copy_exome_to_hdfs_on_gcs.sh [hunk 2]
```diff
@@ -3,8 +3,8 @@
 # Copy exome data to HDFS on a GCS cluster.
 
 ${GATK_HOME:-../..}/gatk-launch ParallelCopyGCSDirectoryIntoHDFSSpark \
```
Comment (sooheelee) [inline scripts/spark_eval/copy_exome_to_hdfs_on_gcs.sh]: Remember the engine team will change `gatk-launch` to `gatk` soon.

File: src/main/java/org/broadinstitute/hellbender/tools/spark/ParallelCopyGCSDirectoryIntoHDFSSpark.java [hunk 3]
```diff
@@ -26,17 +26,47 @@
 import java.util.stream.Collectors;
 
 /**
- * This tool uses Spark to do a parallel copy of either a file or a directory from GCS into HDFS.
- * Files are divided into chunks of size equal to the HDFS block size (with the exception of the final
- * chunk) and each Spark task is responsible for copying one chunk. To copy all of the files in a GCS directory,
- * provide the GCS directory path, including the trailing slash. Directory copies are non-recursive so
+ * Stage a Google Cloud Storage file or directory into an HDFS file system.
```
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/spark/ParallelCopyGCSDirectoryIntoHDFSSpark.java]: ChrisN merged the new program groups this afternoon.
So replace `SparkProgramGroup` with `OtherProgramGroup` after rebasing.

File: src/main/java/org/broadinstitute/hellbender/tools/spark/ParallelCopyGCSDirectoryIntoHDFSSpark.java [hunk 4]
```diff
@@ -26,17 +26,47 @@
...
+ * </ul>
  */
 @DocumentedFeature
-@CommandLineProgramProperties(summary="Parallel copy a file or directory (non-recursive) from GCS into HDFS",
-        oneLineSummary="Parallel copy a file or directory from GCS into HDFS.",
+@CommandLineProgramProperties(summary="This tool copies a file or directory from Google Cloud Storage into" +
+        "an HDFS file system on a Spark cluster. Data is copied in parallel using multiple Spark tasks in order" +
+        "to enable a high-throughput transfer.",
+        oneLineSummary="Parallel copy a file or directory from Google Cloud Storage into HDFS",
         programGroup = SparkProgramGroup.class)
```
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/spark/ParallelCopyGCSDirectoryIntoHDFSSpark.java]: replace `SparkProgramGroup` with `OtherProgramGroup` after rebasing

**Comment by sooheelee** [comment]
@droazen I had put in https://github.com/broadinstitute/gatk/issues/3899 over a week ago and just now assigned you to decide if we want to consider instituting a symlink/environmental variable `gatk` that is callable from anywhere in the Docker. 

As someone with a newbie perspective, it is easier for me to grasp `gatk` represents the script to which I must provide the path to (e.g. ~/Downloads/gatk/gatk) than to understand that `./gatk` must be run in a particular folder.

**Comment by sooheelee** [comment]
What will happen to this PR @cwhelan given @tedsharpe is covering the tools already?

**Comment by droazen** [comment]
Rebased and updated -- will merge once tests pass.

**Pull request merged**